packages/runtime/lib/generator/runtime-generator.js

```diff
@@ -453,6 +452,20 @@ class RuntimeGenerator extends BaseGenerator {
       contents: JSON.stringify(currrentPackageJson, null, 2)
     })
 
+    // set new entrypoint if specified
```

Comment:
OK, so the `setEntryPoint` on generator for update, will be useless, right?

Reply:
yes, just call `update({ services: [...], entrypoint: 'foobar' })`